app/assets/css/bootstrap-override.css

```diff
@@ -70,11 +70,19 @@
 }
 
 .switch input[type='checkbox']:disabled + .slider {
-  background-color: var(--ui-gray-3);
-  @apply th-dark:before:bg-gray-warm-8;
-  @apply th-highcontrast:before:bg-gray-warm-8;
-  @apply th-dark:bg-gray-warm-9;
-  @apply th-highcontrast:bg-gray-warm-9;
+  opacity: 0.3;
+}
+
+.switch input[type='checkbox']:disabled + .slider:is([theme='dark'] *),
+.switch input[type='checkbox']:disabled + .slider:is([theme='highcontrast'] *) {
+  opacity: 0.4;
+  background-color: var(--ui-gray-warm-6);
+}
+
+.switch input[type='checkbox']:disabled:is(:checked) + .slider:is([theme='dark'] *),
+.switch input[type='checkbox']:disabled:is(:checked) + .slider:is([theme='highcontrast'] *) {
+  opacity: 0.6;
+  background-color: var(--ui-blue-9);
 }
```

Comment:
having the colors referenced directly here means I don't have to search far and wide in theme.css to understand what color is applied or to change the color in future

Reply:
```suggestion
  opacity: 0.3;
  @apply th-dark:bg-gray-warm-6/40;
  @apply th-highcontrast:bg-gray-warm-6/40;
}

.switch input[type='checkbox']:disabled:is(:checked) {
  @apply th-dark:bg-blue-9/60;
  @apply th-highcontrast:bg-blue-9/60;
}
```